src/type/introspection.js

```diff
@@ -16,7 +16,7 @@ import {
   GraphQLEnumType,
   GraphQLInputObjectType,
   GraphQLList,
-  GraphQLNonNull
+  GraphQLNonNull,
```

Comment:
don't we like trailing commas?

I added an import during the process of doing stuff. Didn't feel compelled to remove it.